host_vars/ak36-gw/gateway.yml

```diff
@@ -77,6 +77,7 @@ tunnel_metric: 1024
 # Tunnel metric 1024 as most internet uplinks will hardly reach 40MBit/s
 
 wireguard_base_prefix: 2001:bf7:750:4002::/64
+wireguard_base_v4prefix: 10.31.145.0/24
```

Comment:
why dont set that on saarbruecker and ohlauer?

Reply:
Probably I made a mistake doing a rebase.

Reply:
I did this.